README.md

```diff
@@ -23,6 +23,20 @@ In addition to [standard `story-json` properties](https://github.com/micnews/sto
 | `analytics` | Array of [AMP analytics objects](https://www.ampproject.org/docs/reference/components/amp-analytics) |
 | `bookendConfigSrc` | [Bookend endpoint URL](https://www.ampproject.org/docs/tutorials/visual_story/create_bookend) |
 
+### Google AMP specific fields in `story-json`
+
+- In `story-json` attribute `meta.authorType` can be `Organization` or `Person`. This is uses in structured data.
+-  In `story-json` attribute `meta.images` is an array of images. This is used for AMP story metadata as well as structured data for SEO.
+      where 
+      - first one is square 
+      - second one is `poster-portrait`
+      - third one is `poster-landscape`
```

Comment:
@pietrop what's the reason for using array vs named keys? i.e `images: { square: ..., landscape: ... }`?

Reply:
Was following the structured data specs, where they are added to the json as an array. 

- https://developers.google.com/search/docs/data-types/article#properties
- https://search.google.com/structured-data/testing-tool

Reply:
I think this can be confusing as we require images to be in the specific order. From the doc looks like structured data schema allows different images to be in any order.